### ISSUE TITLE
Accessibiity failure:  Understanding SC 2.1.1 Keyboard (Level A)

### ISSUE THREAD

**Issue by PapaSierra555** [queen-of-code/alexa-recipe-app#77]
Adding a new picture presents a Replace Picture link.  Link is not keyboard accessible.  Fails:

Success Criterion 2.1.1 Keyboard 
https://www.w3.org/TR/WCAG22/#keyboard
Keyboard (Level A)
